### DISCUSSION TITLE
Question: What is one thing you'd change about the game?

### DISCUSSION THREAD

**Discussion by aflacc** [FunkinCrew/Funkin#1575] [Q&A]
If you could change one thing that already exists in friday night funkin, what would it be? By this I mean mechanics, concepts, design, or really anything as long as it reworks or changes something that is already in the game.

**Comment by ghost** [comment]
input system

**Comment by isakube** [comment]
tbh i'd add middle scroll as an option

**Comment by ghost** [comment]
get that good ProjectFNF input in there

**Comment by aflacc** [comment]
> tbh i'd add middle scroll as an option

Middle scroll is cool but I feel like it could mess with a few things that could be done in the future

**Comment by GOOBOZA** [comment]
id probably make it so the hit boxes for blue orbs were bigger

**Comment by GOOBOZA** [comment]
and id get circle hit boxes

**Comment by GOOBOZA** [comment]
and id put MGHP6.3 in the main game
